src/components/CharactersPage/CharacterForm/CharacterForm.js

```diff
@@ -71,6 +78,34 @@ const CharacterForm = ({ onClose }) => {
     reValidateMode: "onSubmit",
   })
 
+  useEffect(() => {
+    reset({
+      name: character?.name || "",
+      eyeColor: character?.eyeColor || "",
```

Comment:
You should reset your form when we have a character, think about how you can simplify this to achieve that